lib/lambda_ethereum_consensus/validator/duties.ex

```diff
@@ -27,26 +26,38 @@ defmodule LambdaEthereumConsensus.Validator.Duties do
 
   @type proposer_duty :: Types.validator_index()
 
+  @type sync_committee_aggregator_duty :: %{
+          aggregated?: boolean(),
+          selection_proof: Bls.signature(),
+          contribution_domain: Types.domain(),
+          subcommittee_index: Types.uint64()
+        }
+
   @type sync_committee_duty :: %{
-          # Given that we send messages in EVERY slot, instead
-          # of tracking them all, we just keep track of the last
-          # slot we broadcasted to avoid double publishs.
-          last_slot_broadcasted: Types.slot(),
+          broadcasted?: boolean(),
+          message_domain: Types.domain(),
+          validator_index: Types.validator_index(),
           subnet_ids: [Types.uint64()],
-          validator_index: Types.validator_index()
+          aggregation: [sync_committee_aggregator_duty()]
         }
 
+  @type misc_data :: %{sync_subcommittee_participants: %{}}
```

Comment:
Probably bad naming (misc was from miscellaneous), maybe shared or common? The idea is to have a place for common cached calculations not associated to a particular slots/duty, they could be recalculated at will (every epoch, every period, etc) and may be a way to avoid recalculating costly operations, for example, the map we were discussing regarding validator indices could be placed here as well

Reply:
I moved on with `shared` but added the particular `sync_subcommittee_participant` functions as discused previously.

Reply:
Also added a comment related to this type to be clear of the intentions.